tests/models/test_initialization.py

```diff
@@ -11,6 +12,9 @@
 @pytest.mark.parametrize("model_arch", HF_EXAMPLE_MODELS.get_supported_archs())
 def test_can_initialize(model_arch):
     model_info = HF_EXAMPLE_MODELS.get_hf_info(model_arch)
+    if (model_arch == "Cohere2ForCausalLM"
+            and transformers.__version__ < "4.48.0"):
```

Comment:
Why is this needed

Reply:
The model is only available in transformers v4.48 which has not been released yet.

Reply:
Thanks. That makes sense.